Sources/TuistGraph/Models/SwiftPackageManagerDependencies.swift

```diff
@@ -10,4 +10,58 @@ public struct SwiftPackageManagerDependencies: Equatable {
     public init(_ packages: [Package]) {
         self.packages = packages
     }
+
+    /// Returns `Package.swift` representation.
+    ///
+    /// **NOTE** It is a temporary solution until Apple resolves: https://forums.swift.org/t/pitch-package-editor-commands/42224
+    public func manifestValue(swiftVersion: String) -> String {
+        """
+        // swift-tools-version:\(swiftVersion)
+
+        import PackageDescription
+
+        let package = Package(
+            name: "PackageName",
+            dependencies: [
+                \(packages.map { $0.manifestValue + "," }.joined(separator: "\n        "))
```

Comment:
wouldn't it be better to use `\t` instead of multiple spaces?

Reply:
I replaced it but in my opinion tests are less readable now.